server/controllers/authController.js

```diff
@@ -1,8 +1,19 @@
-const User = require('../models/user')
+const User = require('../models/user');
 
 const handleErrors = (err)=>{
     let errors = {email: '', password: ''};
-
+    //login 
+       //incorrect email
+       if(err.message === "incorrect email"){
+           errors.email = 'Sorry email is not registered';
```

Comment:
I think it would be better if you use the same error message maybe like **Incorrect email or password.**. For security purposes, we don't need the user to know the exact missing or incorrect credentials.

Reply:
alright i'll fix it like that